sdks/java/io/common/src/test/java/org/apache/beam/sdk/io/common/IOTestPipelineOptions.java

```diff
@@ -88,4 +88,10 @@
   Integer getCassandraPort();
   void setCassandraPort(Integer port);
 
+  /* Text */
+  @Description("Lines of text count")
```

Comment:
<!--new_thread; commit:c6c7070ad92424707d3720d3a4dc2c0fb6961440; resolved:0-->
Number of lines of text that will be written and read by the test.

Reply:
Ok  - will modify accordingly to a new property name (the generic one)